docs/specification/1.0.md

```diff
@@ -441,46 +441,61 @@ The EHR’s authorization server is responsible for enforcing restrictions on th
+The authorization server’s response MUST indicate that the access token MAY NOT be cached or otherwise persisted in data storage.  The expiry time for which the token remains value SHOULD BE set as short as possible to provide the requested decision support.
 
 ```json
 {
   "fhirAuthorization" : {
     "access_token" : "some-opaque-fhir-access-token",
-    "token_type" : "Bearer",
+    "token_type" : "bearer",
     "expires_in" : 300,
-    "scope" : "patient/Patient.read patient/Observation.read"
+    "scope" : "patient/Patient.read patient/Observation.read",
+    "subject" : "cds-service4",
+    "aud" : "http://fhirServer.ehr.com
   }
 }
 ```
 
-#### Access Token
+#### Passing the Access Token to the CDS Service
 
-The access token is specified in the CDS Service request via the `fhirAuthorization` request parameter. This parameter is an object that contains both the access token as well as other related information.
+The access token is specified in the CDS Service request via the OPTIONAL `fhirAuthorization` request parameter. This parameter is an object that contains both the access token as well as other related information.  If the EHR chooses not to pass along an access token, the `fhirAuthorization` parameter is omitted.  
 
-Field | Description
------ | -----------
-`access_token` |*string*. This is the OAuth 2 access token that provides access to the FHIR server.
-`token_type`   |*string*. Fixed value: `Bearer`
-`expires_in`   |*integer*. The lifetime in seconds of the access token.
-`scope`        |*string*. The scopes the access token grants the CDS Service.
-
-It is recommended that the `expires_in` value be very short lived as the access token must be treated as a transient value by the CDS Service.
-
-It is recommended that the `scope` value contain just the scopes that the CDS Service needs for its logic and no more.
-
-As the CDS Service is executing on behalf of a user, it is important that the data the CDS Service has access to is under the same restrictions/authorization as the current user. As such, the access token shall be scoped to:
-
-- The CDS Service being invoked
-- The current user
+Field | Priority | Description
+----- | ----- | -----------
+`access_token` | REQUIRED | *string*. This is the OAuth 2 access token that provides access to the FHIR server.
+`token_type`   | REQUIRED | *string*. Fixed value: `Bearer`
+`expires_in`   | REQUIRED | *integer*. The lifetime in seconds of the access token.
+`scope`        | REQUIRED | *string*. The scopes the access token grants the CDS Service.
+`subject` | REQUIRED | *string*.  The CDS Service as registered with the EHR.
```

Comment:
Would "The OAuth 2 client identifier of the CDS Service" be a better description of `subject`?